teammapper-frontend/src/assets/i18n/it.json

```diff
@@ -166,7 +166,8 @@
     "NODE_COPIED": "Nodo copiato con successo!",
     "ERRORS": {
       "ROOT_NODE_COPIED": "Non è possibile copiare il nodo radice!",
-      "GENERIC_NODE_COPY_ERROR": "Si è verificato un errore durante il tentativo di copiare il nodo radice."
+      "GENERIC_NODE_COPY_ERROR": "Si è verificato un errore durante il tentativo di copiare il nodo radice.",
```

Comment:
as in the other PR, suggestion regarding naming of translations: `NODE_COPY_GENERIC_ERROR` or something

Reply:
This has been done in a separate PR: https://github.com/b310-digital/teammapper/pull/330